src/admin-panel/index.js

```diff
@@ -0,0 +1,172 @@
+const selectElement = document.getElementById('select-tags');
+const main = document.getElementById('main');
+const BASE_URL = 'https://api.realdevsquad.com';
+
+function createTagCreationComponent() {
+  const title = createElement({
+    type: 'h2',
+    attributes: { class: 'title' },
+    innerText: 'Create Skill Tags',
+  });
+  const tagCreationContainer = createElement({
+    type: 'div',
+    attributes: { class: 'tag-create__container' },
+  });
+  const tagNameInput = createElement({
+    type: 'input',
+    attributes: {
+      class: 'input',
+      placeholder: 'Enter Skill Name',
+      id: 'skill',
+    },
+  });
+  const submit = createElement({
+    type: 'button',
+    attributes: { class: 'submit-button' },
+    innerText: 'Create Skill',
+  });
+  submit.addEventListener('click', createSkill);
+  tagCreationContainer.appendChild(title);
+  tagCreationContainer.appendChild(tagNameInput);
+  tagCreationContainer.appendChild(submit);
+  return tagCreationContainer;
+}
+
+async function createLevel() {
+  try {
+    const inputValue = document.getElementById('level').value;
+    const button = document.querySelector('.submit-button');
+    const body = {
+      name: inputValue,
+      levelnumber: inputValue,
+    };
+    setLoadingState();
+    button.textContent = 'Creating Level...';
+
+    const response = await fetch(`${BASE_URL}/levels`, {
+      method: 'POST',
+      credentials: 'include',
+      body: JSON.stringify(body),
+      headers: {
+        'Content-type': 'application/json',
+      },
+    });
+    const data = await response.json();
+    const { message } = data;
+    alert(message);
+  } catch (err) {
+    alert(err);
+  } finally {
+    removeLoadingState();
+    button.textContent = 'Create Level';
+  }
+}
+
+async function createSkill() {
+  try {
+    const inputValue = document.getElementById('skill').value;
+    const button = document.querySelector('.submit-button');
+
+    const body = {
+      name: inputValue,
+      type: 'SKILL',
+      reason: 'adding skills to users',
+    };
+
+    setLoadingState();
+    button.textContent = 'creating skill...';
+
+    const response = await fetch(`${BASE_URL}/tags`, {
+      method: 'POST',
+      credentials: 'include',
+      body: JSON.stringify(body),
+      headers: {
+        'Content-type': 'application/json',
+      },
+    });
+
+    const data = await response.json();
+    const { message } = data;
+    alert(message);
+  } catch (err) {
+    alert(err);
+  } finally {
+    removeLoadingState();
+    button.textContent = 'Create Skill';
+  }
+}
+
+function createLevelSelector() {
+  const levelSelector = createElement({
+    type: 'select',
+    attributes: { class: 'level-selector' },
+  });
+
+  for (let i = 0; i <= 5; i++) {
+    const option = createElement({
+      type: 'option',
+      attributes: { class: 'option', value: i },
+      innerText: i,
+    });
+    levelSelector.append(option);
+  }
+  return levelSelector;
```

Comment:
NIT: appending elements can be done after loop is completed, and The times for loop runs should come from a constant